Automattic/Edit-Flow
# Fix custom status error with Gutenberg 7.2+

A fix for #583. Continuing to break the rules, but for a good cause. 

Props to @ryelle for tracking this down

## Steps to Test

1. With Gutenberg 7.2 or 7.3
2. With the custom status module active
3. Try to edit a draft post
4. Editor should not crash
